src/hooks/v1/transactor/V1SetOperatorTx.ts

```diff
@@ -0,0 +1,37 @@
+import { useContext } from 'react'
+import { NetworkContext } from 'contexts/networkContext'
+import { V1UserContext } from 'contexts/v1/userContext'
+
+import { TransactorInstance } from '../../Transactor'
+
+type OperatorData = {
+  operator: string
+  domain: number // projectId (or 0, for any project)
+  permissionIndexes: number[]
+}
+
+export function useV1SetOperatorTx(): TransactorInstance<OperatorData> {
```

Comment:
We'll use this in the UI for https://github.com/jbx-protocol/juice-interface/issues/1335